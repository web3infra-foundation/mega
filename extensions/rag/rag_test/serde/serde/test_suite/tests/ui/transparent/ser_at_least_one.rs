use serde_derive::Serialize;

#[derive(Serialize)]
#[serde(transparent)]
struct S {
    #[serde(skip)]
    a: u8,
}

fn main() {}
