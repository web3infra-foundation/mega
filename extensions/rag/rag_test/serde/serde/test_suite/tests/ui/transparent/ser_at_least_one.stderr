error: #[serde(transparent)] requires at least one field that is not skipped
 --> tests/ui/transparent/ser_at_least_one.rs:4:1
  |
4 | / #[serde(transparent)]
5 | | struct S {
6 | |     #[serde(skip)]
7 | |     a: u8,
8 | | }
  | |_^
